mtags/src/main/scala/scala/meta/internal/pc/SemanticTokens.scala

```diff
@@ -41,4 +41,20 @@ object SemanticTokens {
     SemanticTokenModifiers.DefaultLibrary
   )
 
+  val getTypeId: Map[String, Int] = TokenTypes.zipWithIndex.toMap
+  val getModifierId: Map[String, Int] = TokenModifiers.zipWithIndex.toMap
+
+  val getTypePriority: Map[Int, Int] = Map(
+    0 -> 7, // SemanticTokenTypes.Namespace,
```

Comment:
Any possibility to have these number generated via for example  `TokenTypes.indexOf(SemanticTokenTypes.Namespace)`

Reply:
Here is the problem, the priorities we want to get (I am trying to make it look as in first semantic tokens PR) don't align with `TokenTypes`, which need to be indexed the same as in lsp. I will add comment explaining why we want this order for every token with examples, hopefully it will be clear

Reply:
Sure, but I was thinking more along the lines of:
```suggestion
    0 -> TokenTypes.indexOf(SemanticTokenTypes.Namespace),
```

Reply:
Oh sure, good idea